Orange/widgets/unsupervised/owkmeans.py

```diff
@@ -423,6 +429,8 @@ def set_data(self, data):
             self.run()
 
     def send_report(self):
+        # False positives (Setting is not recognized as int)
+        # pylint: disable=invalid-sequence-index
```

Comment:
I prepared two more commits (the last is not yet completed and pushed), and this remains only in two places, so I'd prefer being conservative wrt disabling warnings.